dds_web/api/files.py

```diff
@@ -304,7 +304,7 @@ def get(self):
                 )
 
             # Check if to return file size
-            show_size = flask.request.args.get("show_size")
+            show_size = flask.request.args.get("show_size",type=inputs.boolean, default=False)
```

Comment:
This is done because I realise that this is the proper way to parse input. Being such a small change I added it here despite it being for files. 
If deemed not usefull to have it here. I can create a separate PR just for this